docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,414 @@
+{
```

Comment:
<p>I would also mention that the reduced-rank coding is used by default because a full-rank coding implies a structural multicollinearity between the dummies and the intercept (i.e., the sum of the 4 dummies would be equal to the intercept). Dropping the intercept gives us the freedom to add the extra dummy. However, the next categorical variable we add must have reduced-rank coding. If not, you have the same problem with structural multicollinearity.</p><p> </p><p>And some notes about formulae. By default, when it detects a categorical variable, levels are not sorted according to appearance, but according to an alphabetic ordering (as in R and Patsy). But if you pass a pandas.DataFrame where the categorical variable has a specific order, it is respected.</p><p> </p><p>The paragraph describing group specific effects is perfect.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/302/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>I think maybe you can improve this section in a future PR. You are going to do a better job than me.</p>

Reply:
Right, there's no problem in leaving it as it is right now. After all, nothing of what it says is wrong!